### ISSUE TITLE
StatementExplain::explainQueryPlan implementation incorrect

### ISSUE THREAD

**Issue by dourgulf** [Tencent/wcdb#837]
### What's the issue?
```Objective-C
StatementExplain &StatementExplain::explainQueryPlan(const Statement &statement)
{
    m_description.append("EXPLAIN QUERY PLAN" + statement.getDescription());
    return *this;
}
```
missed a space after "PLAN", it should be "PLAN "

**Comment by cqwicqw** [comment]
Got it.

**Comment by Qiuwen-chen** [comment]
Fixed in WCDB 2.0
